sdk/src/main/java/com/hcaptcha/sdk/tasks/Task.java

```diff
@@ -92,6 +95,19 @@ protected void setException(@NonNull HCaptchaException hCaptchaException) {
         tryCb();
     }
 
+    /**
+     * Internal callback which called once 'open-callback' fired in js SDK
+     */
+    protected void captchaOpened() {
+        final Iterator<OnOpenListener> iterator = onOpenListeners.iterator();
+        while (iterator.hasNext()) {
+            final OnOpenListener onSuccessListener = iterator.next();
+            onSuccessListener.onOpen();
+            // Remove listener as result was successfully delivered
+            iterator.remove();
```

Comment:
Listener should not be removed as it may be triggered multiple times.

Reply:
Agree, done